appd/README.md

```diff
@@ -0,0 +1,141 @@
+# Improved Cosmos SDK Upgrading System
+
+This repository contains an improved Cosmos SDK Upgrading system (codename: `nova`).
+Sync from genesis or upgrade a Cosmos SDK chain seemlessly for the node operators, without binary switch.
+
+## Features
+
+- **Automatic Upgrading**: The system automatically upgrades the chain to the latest version of the software after an `AppVersion` change.
+- **No binary switch required**: At the upgrade height, no binary switch is required, a huge improvement compared to solely relying on the `x/upgrade` module.
+- **One binary**: The system uses only one binary for all upgrades, which simplifies  **syncing from genesis**, the upgrade process and reduces the risk of errors.
+- **Passthrough mode**: The system can be used in passthrough mode, where a user can interact with an older binary directly.
+- **Speed of native**: `Nova` is designed to minimize overhead and is only used for the duration of the upgrade process. Once the upgrade is complete, the system use the native app.
+
+## How it works
+
+The system works by running embedded the previous consensus breaking version within the binary and running a single CometBFT instance for all chains.
```

Comment:
we mention embedding in the nova docs, but this actually happens at the app level, nova doesn't care at all about anything being embedded, it just requires a `Version` with bytes. It might nice to make this distinction and also show a code example of embedding a binary WDYT?